quickstart/mutators.markdown

```diff
@@ -758,9 +765,134 @@ The switch mutator finds the first label within a switch statement that differs
+
+| Original operator | AOR_1 | AOR_2 | AOR_3 | AOR_4 |
+|-------------------|-------|-------|-------|-------|
+| \+                | \-    | *     | /     | %     |
+| \-                | \+    | *     | /     | %     |
+| *                 | /     | %     | \+    | \-    |
+| /                 | *     | %     | \+    | \-    |
+| %                 | *     | /     | \+    | \-    |
+{:.table}
+
+<a name="AOD" id="EXPERIMENTAL_AOD"></a>
 
-1. <a name="fn1" id="fn1"></a> Integer numbers and booleans are actually represented in the same way by the JVM,
+Arithmetic Operator Deletion Mutator (AOD)
+-------------------------------------------------
+This mutator replaces an arithmetic operation with one of its members. The mutator is composed of 2 sub-mutators, AOD_1 and AOD_2, that mutate the operation to its first and second member respectively.
+For example
+
+```java
+int a = b + c;
+```
+
+will be mutated to
+
+```java
+int a = b;
+```
+
+and to
+
+```java
+int a = c;
+```
+
+<a name="CRCR" id="EXPERIMENTAL_CRCR"></a>
+
+Constant Replacement Mutator (CRCR)
+-------------------------------------------------
+Like the inline constant mutator, this mutator mutates inline constant. The mutator is composed of 6 sub-mutators (CRCR1 to CRCR6) that mutate constants according to the table below.
+
+| Constant | CRCR1 | CRCR2 | CRCR3 | CRCR4 | CRCR5 | CRCR6 |
+|----------|-------|-------|-------|-------|-------|-------|
+| c        | 1     | 0     | -1    | -c    | c+1   | c-1   |
+{:.table}
+
+<a name="OBBN" id="EXPERIMENTAL_OBBN"></a>
+
+Bitwise Operator Mutator (OBBN)
+-------------------------------------------------
+This mutator mutates bitwise and (&) and or (|). It is composed of three sub-mutators, OBBN1, OBBN2 and OBBN3 that respectively reverse the operators, replace a bitwise operation by its first member, and by its second member.
+For example
+
+```java
+a & b;
+```
+
+will be mutated to
+
+```java
+a | b;
+```
+
+by OBBN1, to
+
+```java
+a;
+```
+
+by OBBN2 and to
+
+```java
+b;
+```
+
+by OBBN3.
+
+<a name="ROR" id="EXPERIMENTAL_ROR"></a>
+
+Relational Operator Replacement Mutator (ROR)
+-------------------------------------------------
+This mutator replaces a relational operator with another one. The mutator is composed of 5 sub-mutators (ROR1 to ROR5) that mutate the operators according to the table below.
+
+| Original operator | ROR1 | ROR2 | ROR3 | ROR4 | ROR5 |
+|-------------------|------|------|------|------|------|
+| &lt;              | &lt;=| &gt; | &gt;=| ==   | !=   |
+| &lt;=             | &lt; | &gt; | &gt;=| ==   | !=   |
+| &gt;              | &lt; | &lt;=| &gt; | ==   | !=   |
+| &gt;=             | &lt; | &lt;=| &gt; | ==   | !=   |
+| ==                | &lt; | &lt;=| &gt; | &gt;=| !=   |
+| !=                | &lt; | &lt;=| &gt; | &gt;=| ==   |
```

Comment:
It's possible (better I think) to use `<` for `&lt;` and `\>` got `&gt;` as it was done for the CONDITIONALS_BOUNDARY table :
```
| Original conditional | Mutated conditional |
|----------------------|---------------------|
| <                    | <=                  |
| <=                   | <                   |
| \>                   | \>=                 |
| \>=                  | \>                  |
{:.table }
```